docs/the-new-architecture/use-app-template.md

```diff
@@ -1,7 +1,130 @@
 ---
 id: use-app-template
-title: Using the New App Template
+title: Creating a New Architecture App
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+import RemoveGlobalCLI from '.././\_remove-global-cli.md';
+
+This page will help you create a new React Native app that uses the New Architecture.
+
+## Development environment
+
+Before continuing, make sure you've followed all the steps in [Setting up the development environment](getting-started.md), under the **React Native CLI Quickstart** tab.
+
+If following the setup guide, stop when you reach the section **Running your React Native Application**, and resume following this guide.
+
+:::note
```

Comment:
```suggestion
:::caution
```

Reply:
For my future reference, is there a particular reason you want to go with `caution` here? I went with `note` for any that just add extra info, since it felt like a lot of yellow blocks on the page otherwise, very visually busy.

Reply:
IIRC, note/info box are blue and white. They can slip and the user may not notice them. I think that they are good for additional information that, even if the users don't read them, the output doesn't change very much.

A caution box is yellow, and harder to miss. I think we should use it every time we are delivering information that, if missed, may generate some trouble. For example, if the users configure the New Architecture on version 0.67, they will incur in a lot of errors. And it could be tricky to spot that they are using the wrong version. Hence, the yellow box.
Similarly, if the skipped the `Settings Up Your Environment` guide because they are used to work with Expo and they try to use the New Architecture, it won't work, and a yellow box could catch their attention more easily.

Does this make sense?

Reply:
I think I have a different opinion about how this would be read -- for me, too many yellow boxes can visually interfere with the ability to skim the page's structure -- but it's a matter of opinion, so happy to stick with this.